src/codemodder/code_directory.py

```diff
@@ -1,65 +1,79 @@
-from pathlib import Path
-
-
-DEFAULT_INCLUDE = "*.py"
-
-
-def py_file_matches(file_path, patterns):
-    """
-    file_path is suffixed with `.py` and matches at least one pattern.
-    """
-    is_py_file = file_path.match(DEFAULT_INCLUDE)
-    if not patterns:
-        return is_py_file
-    return is_py_file and any(
-        file_path.match(pattern.rsplit(":")[0]) for pattern in patterns
-    )
+from typing import Optional, Sequence
 
+import fnmatch
+import itertools
+from pathlib import Path
 
-def file_line_patterns(file_path, patterns):
+DEFAULT_INCLUDED_PATHS = ["**.py", "**/*.py"]
```

Comment:
I believe the second glob pattern includes the files from the first one.

Reply:
good test case to add somehow bc I thought the same thing. Honestly the best thing about this PR is adding fnmatch!